doc/release-notes.md

```diff
@@ -10,6 +10,24 @@ Option handling
 - The `-reindex` and `-reindex-chainstate` options now imply `-rescan`
   (provided that the wallet is enabled and pruning is disabled, and unless
   `-rescan=0` is specified explicitly).
+- A new `-anchorconfirmations` argument has been added to allow the user
+  to specify the number of blocks back from the chain tip that anchors 
+  will be selected from when spending notes. By default, anchors will
+  now be selected to have 10 confirmations. Values
```

Comment:
Since the "base case" of spends is that you're spending untrusted notes I think that we should keep at 10 confirmations for the default, and then allow trusted notes to default to using fewer in the future.

Reply:
After discussion, the decision is to make the anchor require 3 confirmations by default, but set minconf's default to 10. This will mean that extra information is not revealed by the choice of anchor depth.

Reply:
Sounds good to me.